bootstrap/README.md

```diff
@@ -32,7 +32,7 @@ go install github.com/in3rsha/bitcoin-utxo-dump@5723696e694ebbfe52687f51e7fc0ce6
 
 ```shell
 BITCOIN_DIR=/path/to/bitcoin-22.0/
-NETWORK=<mainnet or testnet>
+NETWORK=<mainnet or testnet4>
```

Comment:
Should it still say `testnet4` here?

Reply:
This gets propagated to bitcoin_d so I thought we should not map that.